lib/modules/manager/npm/utils.spec.ts

```diff
@@ -53,5 +53,12 @@ describe('modules/manager/npm/utils', () => {
       const lockFileComposed = composeLockFile(lockFile, '  ');
       expect(lockFileComposed).toMatchSnapshot();
     });
+
+    it('adds trailing newline to match npms behaviour and avoid diffs', () => {
```

Comment:
not entirely sure if this is a good or needed test. Basically it should be covered by the snapshots already, but I wanted to check if a real lockfile generated by npm, without any manual modifications, can be cleanly parsed to an object and composed to a string again without any modifications.